docs/user_guide/mitigation.rst

```diff
@@ -68,6 +68,263 @@ Postprocessing
 
 .. currentmodule:: fairlearn.postprocessing
 
+Fairlearn currently supports one postprocessing technique that is based on the
+paper *Equality of Opportunity in Supervised Learning* [#3]_. Unlike other
+mitigation techniques :class:`ThresholdOptimizer` is built to satisfy the
+specified fairness criteria exactly and with no remaining disparity.
+In many cases this comes at the expense of performance, for example, with
+significantly lower accuracy. Regardless, it is a useful data point to compare
+results with.
+
+We can think of each thresholding rule as a binary classifier and determine
```

Comment:
This description is only true if you select "equalized_odds" as the constraints. Equalized odds are a special case.

I would begin with the simple cases:
* for each sensitive feature, consider classifiers obtained by thresholding its prediction and plot the curve corresponding to the upper envelope of all achievable combinations of the objective and the constraint metric
* simultaneously walk along all the curves--so as to maintain the same value of the constraint--while monitoring the objective value, and pick the combination of thresholds that optimizes the objective
* that's exactly what's depicted if you run `plot_threshold_optimizer`
* To achieve the precise value of the constraint along the curve, you typically need to randomize between two thresholds, hence the need for `p0` and `p1`
* Flip tells you whether you're allowed to flip the monotonicity... the need for flipping may suggest that the underlying classifier is not behaving as expected and any such instance should be inspected.

For equalized odds, we proceed differently:
* Here we again consider all possible threshold rules for each sensitive feature, and plot the achievable set of true positive rate / false positive rate combinations. (I'm not sure that calling it ROC curve is actually helpful here. Is it?)
* We take overlap among all these plots to get the set of TPR/FPR combinations achievable by all sensitive features.
* We select the combination that maximizes the objective (equalized odds will be satisfied automatically).
* The operation of taking an overlap may require that we step below the Pareto curve for a given feature, and it means that with a certain probabily `p_ignore` we may need to toss a biased coin, with the bias indicated by `prediction_constant`

Reply:
I've reordered it as you said and adjusted the description. The original idea was to start with the key idea of the paper (eq. odds postprocessing), but obviously that's less relevant for users. If they want to read the paper they can always do that.